system/Database/SQLSRV/Builder.php

```diff
@@ -290,6 +290,20 @@ private function getFullName(string $table): string
         }
 
         if ($this->db->escapeChar === '"') {
+            if (str_contains($table, '.') && ! str_starts_with($table, '.') && ! str_ends_with($table, '.')) {
+                $dbInfo   = explode('.', $table);
+                $database = $this->db->getDatabase();
+                $table    = $dbInfo[0];
+
+                if (isset($dbInfo[1], $dbInfo[2])) {
```

Comment:
only verify index 2 seems should be enough, explode always get 2 data at least for contain:

```suggestion
                if (isset($dbInfo[2])) {
```


ref https://3v4l.org/6YBtM

Reply:
Done.